CMakeLists.txt

```diff
@@ -53,6 +54,9 @@ include(cmake/Lint.cmake)
 # internal variable for number of ghost cells
 set(NUMBER_GHOST_CELLS ${PARTHENON_NGHOST}) 
 
+set(REGRESSION_GOLD_STANDARD 2 CACHE STRING "Version of gold standard to download and use")
```

Comment:
Should this also be an option so that we can set it at configure time? It might be useful when debugging to backtest these things with older versions of the gold standard.

Reply:
You make a good point, if the version is a cached variable, the hash should be as well and maybe even the url, so that one can test against a local file (url support paths) before uploading.

Reply:
@Yurlungur this already is an option, it's just not using the "option" keyword as it's not a bool variable.
@junghans this is a good point (hash and url). However, from my point of view, this may be more convoluted than just using "local" files (as checked by the "current_version" file content), i.e., instead of letting the user set the version variable, the new url, and the hash, the user just puts arbitrary files in the gold_standard folder and sets a matching string in the "current_version" file.
This way we'd keep the logic to do testing/getting the gold files in a clean from a downstream user point of view.
I updated the docs below to reflect this. Please let me know what you think.

Reply:
The improved docs and this strategy work for me.